src/DevOps/phase-features-with-feature-flags.md

```diff
@@ -0,0 +1,173 @@
+---
+title: DevOps - Phase the features of your application with feature flags
+description: Explore how to phase your application features roll-out using feature flags
+ms.assetid: 371FB731-45F1-4B5F-B9DF-282204CE3E59
+ms.prod: vs-devops-phase-rollout-with-feature-flags
+ms.technology: vs-devops-articles
+ms.manager: willys
+ms.date: 06/20/2017
+ms.author: willys
+author: willys
+---
+
+> 
+> #**THIS IS DRAFT.2 - WORK IN PROGRESS **
+> 
+
+# Phase the features of your application with feature flags
+
+In today's fast-paced, feature-driven markets, it is imperative to be able to deliver value and receive feedback on features quickly and continuously. Collaborating with end-users to get early versions of features vetted out is extremely valuable.
+
+Are you planning to continuously integrate features into your application while they're under development? You probably have a few questions, such as:
+
+- How can you toggle features to hide, disable, or enable features at run-time?
+- How can you revert a change deployed to production without rolling back your release?
+- How can you present users with variants of a feature, to determine which one performs better?
+
+This topic aims to answer this and share our implementation of feature flags (FF) and A|B testing, within our production infrastructure for DevLabs extensions.
+
+## Considerations
+
+Before you introduce feature flags to your engineering process, it's important to consider the following:
+
+- Which users are you planning to target? For example, do you want to target specific users, all users, or both?
```

Comment:
specific users or all users?

sounds odd to have 'both' targeting all and specific, are they not mutually exclusive?